src/d3d9/d3d9_device.cpp

```diff
@@ -443,7 +442,7 @@ namespace dxvk {
       m_autoDepthStencil = nullptr;
     } else {
       // Extended devices only reset the bound render targets
-      for (uint32_t i = 0; i < caps::MaxSimultaneousRenderTargets; i++) {
+      for (uint32_t i = 1; i < caps::MaxSimultaneousRenderTargets; i++) {
```

Comment:
Changing it to 1 here is fine because we don't check for D3DPOOL_DEFAULT resources on extended devices.